agent/util-scripts/pbench-move-results

```diff
@@ -8,8 +8,24 @@ pbench_bin="$(realpath -e ${script_path}/..)"
 . "${pbench_bin}"/base
 
 function usage() {
-    printf "usage:\n"
-    printf "${script_name} [--help] [--controller=<controller>] [--user=<user>] [--prefix=<path>] [--xz-single-threaded] [--show-server]\n"
+    printf -- "usage:\n%s [--help] --user=<user> [--controller=<controller>]" "${script_name}"
+    printf -- " [--prefix=<path>] [--xz-single-threaded] [--show-server]\n"
+    printf -- "\nThe '--user' option value is required, and can also be provided\n"
+    printf -- "by the 'PBENCH_USER' environment variable.\n"
+    printf -- "\nThe '--controller' option is also required, but defaults to\n"
+    printf -- "the 'hostname -f' value, or the 'PBENCH_CONTROLLER' environment\n"
+    printf -- "variable.\n"
+    printf -- "\nThe '--prefix' option allows the user to specify an optional\n"
+    printf -- "directory path hierarchy to be used when displaying the result\n"
+    printf -- "tar balls on the pbench server.\n"
+    local op="move"
+    if [[ "${script_name}" == "pbench-copy-results" ]]; then
+	op="copy"
+    fi
+    printf -- "\nThe '--show-server' will not ${op} any results, but resolve and\n"
+    printf -- "then display the pbench server destination for results.\n"
```

Comment:
`op` would be potentially useful in two other places as well (lines 267 and 285), so I suggest doing this calculation near line 4 and putting the result in a global variable for use in all three places.

Also, I suggest doing it with parameter expansion:
```
op=${script_name#pbench-}
op=${op%-results}
```
(If you're going to stick with exact matches, I would recommend being consistent about the RHS (e.g., compare against `"pbench-move-results"` in all cases), and possibly add some code to verify that `$script_name` has only expected values.)